src/app/pages/manage-organisation/findyouradministrator/findyouradministrator.component.html

```diff
@@ -0,0 +1,28 @@
+<!--<app-sidenav [collapsed]="sideNavVisible$ | async"></app-sidenav>-->
```

Comment:
Rename the file name and folder name to follow the current naming conventions 
ex:- forgot-password-success (include dashes between words)